src/tokens/PAXG.sol

```diff
@@ -0,0 +1,114 @@
+pragma solidity >=0.5.12;
+
+contract PAXG {
+    function add(uint256 x, uint256 y) internal pure returns (uint256 z) {
+        require((z = x + y) >= x, "ds-math-add-overflow");
+    }
+    function sub(uint256 x, uint256 y) internal pure returns (uint256 z) {
+        require((z = x - y) <= x, "ds-math-sub-underflow");
+    }
+    function mul(uint256 a, uint256 b) internal pure returns (uint256) {
+        if (a == 0) {
+            return 0;
+        }
+
+        uint256 c = a * b;
+        require(c / a == b);
+
+        return c;
+    }
+    function div(uint256 a, uint256 b) internal pure returns (uint256) {
+        require(b > 0);
+        uint256 c = a / b;
+
+        return c;
+    }
+    event Approval(address indexed src, address indexed guy, uint256 wad);
+    event Transfer(address indexed src, address indexed dst, uint256 wad);
+    
+    event FeeCollected(address indexed from, address indexed to, uint256 value);
+    event FeeRateSet(
+        uint256 indexed oldFeeRate,
+        uint256 indexed newFeeRate
+    );
+
+    string  public  name = "Paxos Gold";
+    string  public  symbol = "PAXG";
+    uint256 public  decimals = 18;
+    uint256                                            _supply;
+    mapping (address => uint256)                       _balances;
+    mapping (address => mapping (address => uint256))  _approvals;
+
+    uint256 public constant feeParts = 1000000;
+    uint256 public feeRate;
+    address public feeRecipient = 0x74271F2282eD7eE35c166122A60c9830354be42a;       // Doesn't really matter where the fees go
+
+    constructor(uint256 supply) public {
+        _balances[msg.sender] = supply;
+        _supply = supply;
+    }
+
+    function totalSupply() public view returns (uint256) {
+        return _supply;
+    }
+    function balanceOf(address src) public view returns (uint256) {
+        return _balances[src];
+    }
+    function allowance(address src, address guy) public view returns (uint256) {
+        return _approvals[src][guy];
+    }
+
+    function transfer(address dst, uint256 wad) public returns (bool) {
+        return transferFrom(msg.sender, dst, wad);
+    }
+
+    function transferFrom(address src, address dst, uint256 wad)
+        public
+        returns (bool)
+    {
+        if (src != msg.sender) {
+            require(_approvals[src][msg.sender] >= wad, "insufficient-approval");
+            _approvals[src][msg.sender] = sub(_approvals[src][msg.sender], wad);
+        }
+
+        require(_balances[src] >= wad, "insufficient-balance");
+        uint256 _fee = getFeeFor(wad);
+        uint256 _principle = sub(wad, _fee);
```

Comment:
nit: spelling, I think `principal` here

Reply:
Looks like they misspelled in the actual token code.